sdk/storage/azure-storage-file-share/migration_guide.md

```diff
@@ -0,0 +1,1208 @@
+                    Class(es) it belongs to
+                </p>
+            </td>
+            <td width="163" valign="top">
+                <p align="right">
+                    API Name
+                </p>
+            </td>
+            <td width="107" valign="top">
+                <p align="right">
+                    Class(es) it belongs to
+                </p>
+            </td>
+        </tr>
+        <tr>
+            <td width="242" valign="top">
+                <p align="right">
+                    generate_account_shared_access_signature
+                </p>
+            </td>
+            <td width="111" valign="top">
+                <p align="right">
+                    FileService
+                </p>
+            </td>
+            <td width="163" valign="top">
+                <p align="right">
+                    generate_account_sas
+                </p>
+            </td>
+            <td width="107" valign="top">
+                <p align="right">
+                    It’s not a class method.
+                </p>
+                <p align="right">
+                    Just import from azure.storage.fileshare directly
+                </p>
+            </td>
+        </tr>
+        <tr>
+            <td width="242" valign="top">
+                <p align="right">
+                    generate_share_shared_access_signature
+                </p>
+            </td>
+            <td width="111" valign="top">
+                <p align="right">
+                    FileService
+                </p>
+            </td>
+            <td width="163" valign="top">
+                <p align="right">
+                    generate_share_sas
+                </p>
+            </td>
+            <td width="107" valign="top">
+                <p align="right">
+                    It’s not a class method.
+                </p>
+                <p align="right">
+                    Just import from azure.storage.fileshare directly
+                </p>
+            </td>
+        </tr>
+        <tr>
+            <td width="242" valign="top">
+                <p align="right">
+                    generate_file_shared_access_signature
+                </p>
+            </td>
+            <td width="111" valign="top">
+                <p align="right">
+                    FileService
+                </p>
+            </td>
+            <td width="163" valign="top">
+                <p align="right">
+                    generate_file_sas
+                </p>
+            </td>
+            <td width="107" valign="top">
+                <p align="right">
+                    It’s not a class method.
+                </p>
+                <p align="right">
+                    Just import from azure.storage.fileshare directly
+                </p>
+            </td>
+        </tr>
+        <tr>
+            <td width="242" valign="top">
+                <p align="right">
+                    make_file_url
```

Comment:
For `make_file_url` let's say "Url (property)" in the API name column and list all columns in the Class column.